packages/discord.js/src/structures/ReactionCollector.js

```diff
@@ -96,7 +96,7 @@ class ReactionCollector extends Collector {
 
     this.on('remove', (reaction, user) => {
       this.total--;
-      if (!this.collected.some(r => r.users.cache.has(user.id))) this.users.delete(user.id);
+      if (!this.collected.some(users => users.cache.has(user.id))) this.users.delete(user.id);
```

Comment:
```suggestion
      if (!this.collected.some(reaction => reaction.users.cache.has(user.id))) this.users.delete(user.id);
```

Reply:
`reaction` is already declared in the upper scope, I was destructuring `users` there, not sure what happened